app/javascript/shared/components/__tests__/useKeyboardShortcuts.test.jsx

```diff
@@ -18,6 +18,57 @@ describe('Keyboard shortcuts for components', () => {
       expect(shortcut.KeyK).toHaveBeenCalledTimes(1);
     });
     
+    it('should fire a function when longer chained keydown is detected', async () => {
+      const shortcut = {
+        "KeyA~KeyB": jest.fn()
+      };
+
+      renderHook(() =>
+        useKeyboardShortcuts(shortcut, document, { timeout: 1500 }),
+      );
+      fireEvent.keyDown(document, { code: "KeyA" });
+      await new Promise(resolve => setTimeout(() => {
+        fireEvent.keyDown(document, { code: "KeyB" });
+        resolve();
+      },600));
```

Comment:
Refactor (blocking): We want to avoid timeouts in tests as it will increase build times.

```suggestion
      await fireEvent.keyDown(document, { code: "KeyB" });
```

Reply:
This it to test updating the timeout option works. If you feel we don't need to test this functionality we can take it out?

Reply:
If it's to test that, consider adding a very low number for the timeout option since these tests add almost 2 seconds to the test run.